platforms/ios/framework/src/iosPlatform.mm

```diff
@@ -147,10 +148,10 @@ void initGLExtensions() {
         return 0;
     }
 
-    TGHttpHandler* httpHandler = [mapView httpHandler];
+    id<TGURLHandler> urlHandler = mapView.urlHandler;
```

Comment:
After working with `protocol` and `id` in another context, I find it so much cleaner for code architecture.